project-aleph-zero.yaml

```diff
@@ -75,4 +75,19 @@ dataSources:
           kind: substrate/EventHandler
           filter:
             module: system
-            method: Remarked
\ No newline at end of file
+            method: Remarked
+        - handler: handleProxyEvent
+          kind: substrate/EventHandler
+          filter:
+            module: proxy
+            method: ProxyAdded
+        - handler: handleProxyRemovedEvent
+          kind: substrate/EventHandler
+          filter:
+            module: proxy
+            method: ProxyRemoved
+        - handler: handleProxyRemovedEvent
+          kind: substrate/EventHandler
+          filter:
+            module: proxy
+            method: PureRemoved            
```

Comment:
Are you shure for pureRemoved?

only has PureKilled:
https://github.com/novasamatech/polkadot-sdk/blob/e1026d7ee22a593cf566a99484eee02a03ecc236/substrate/frame/proxy/src/tests.rs#L600